.github/workflows/desktop-build.yml

```diff
@@ -254,10 +254,10 @@ jobs:
             make_package: 1
             package_suffix: "-macOS13_Intel"
             artifact_name: macOS13_Intel-package
-            qt_version: 6.6.*
+            qt_version: 6.6.3
```

Comment:
before we specified the * to always use the latest point release, what's the reason for pinning it?

Reply:
I'd prefer we find the correct path so we don't have to up this for every point release, which generally bring important bug fixes